### DISCUSSION TITLE
客户端API接口文档错误 || Client API interface documentation error

### DISCUSSION THREAD

**Discussion by L-Super** [walinejs/waline#2165] [Q&A]
这两个接口文档描述一样
https://waline.js.org/reference/client/api.html#commentcount
https://waline.js.org/reference/client/api.html#pageviewcount

---
The descriptions of these two interface documents are the same
https://waline.js.org/reference/client/api.html#commentcount
https://waline.js.org/reference/client/api.html#pageviewcount

**Comment by lizheming** [comment]
这两个接口确实比较类似，接口参数应该是差不多的

---
These two interfaces are indeed similar, and the interface parameters should be similar
